community/contribution.md

```diff
@@ -15,28 +15,34 @@ _This document explains how Axelerant defines contributions with respect to KPIs
 
 An open source community contribution is defined as any piece of documentation (code or otherwise) or any offline activity that is documented and available in the public space for other community members to use.
 
+### Axelerant’s intent
+
+While we understand the above is an open source contribution **Axelerant’s intent is to encourage more meaningful contributions for the community & for the organisation.**
+
+## Important Notes before you continue reading:
```

Comment:
As far as I am concerned, all these are **sidenotes** and points 3 & 4 don't need a special segment (my goal is to keep this as simple as possible). Since I had to point to them over and over, I realised they need to be more prominently placed & maybe re-titled to give more weight. I understand the title needs rewording but it's only an alternative for 'sidenote'. So yes, I can reword the title, but it doesn't make sense for me to put it anywhere but right before the primary section.